Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerEditorServerBus.h

```diff
@@ -60,6 +60,8 @@ namespace Multiplayer
         //! Notification when the Editor multiplayer play mode is over; therefore ending the multiplayer simulation.
         virtual void OnPlayModeEnd() {}
 
+        //! Notification when the server process launched by the editor has crashed (no longer running).
```

Comment:
nit: Do we know its crashed or just terminated with error/disconnected/not responding? Just seeing if name accurately reflects intent or it needs to be slightly more generic to convey meaning.

Reply:
True! It could have either crashed or the user might have accidently stopped the process by hand. Updated from crashed to "stopped unexpectedly"